bundle/manifests/aws-load-balancer-operator.clusterserviceversion.yaml

```diff
@@ -457,6 +468,9 @@ spec:
   provider:
     name: Red Hat Inc.
     url: https://redhat.com
+  relatedImages:
+  - image: quay.io/aws-load-balancer-operator/aws-load-balancer-controller@sha256:d8b5e9a91aca2a4a4de7f9bd2b614c5ba3d4cc62fa8967e94e9539fd7c1940a9
+    name: controller
```

Comment:
I have no idea why `make bundle` updated this.

Reply:
That's a related image discovery mechanism which was added in [v1.20.0](https://github.com/operator-framework/operator-sdk/releases/tag/v1.20.0). Since you used the latest (`v1.33.0`) operator-sdk it filled the related images field of CSV. Overall I think it's a good thing however I'm wondering how you got into this situation as `make bundle` was supposed [to download you the `v1.17.0` version](https://github.com/openshift/aws-load-balancer-operator/blob/main/Makefile#L69) of operator-sdk.

I see 2 ways of how we can move on:
- you can regenerate the bundle using the version from `Makefile` (`mkae bundle` should do all on its own)
- you can update `Makefile` with the latest operator-sdk and put it as a separate commit in your PR

Reply:
I generally work on a mac. The make target for `operator-sdk` wasn't going to work correctly as it downloads the linux_amd64 version. The target does a test if the bin exists in path, so I simply installed it via brew. Of note, this target will redownload the bin every time if it's not in your path.

https://github.com/openshift/aws-load-balancer-operator/blob/1a2c320b27d4b5bb1a9c259db0796c462a11cc98/Makefile#L224

Happy to adjust however you'd like.

Reply:
`Makefile` is not fully Mac friendly indeed. It would be nice if you could do the same as for `opm` target: [code](https://github.com/openshift/aws-load-balancer-operator/blob/1a2c320b27d4b5bb1a9c259db0796c462a11cc98/Makefile#L255-L270). That should cover both of your points: Mac downloads, avoid downloads when the binary is present in `bin/`.

Reply:
Implemented and regenerated bundle using version specified in Makefile.